pyfr/libs/conduit.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
```

Comment:
Import block order is Python standard library, external libraries, PyFR libraries.